name-registry/tests/functions/expiry.rs

```diff
@@ -0,0 +1,41 @@
+mod success {
+    use crate::utils::{
+        abi::{expiry, extend, register},
+        get_contract_instance,
+    };
+    use fuels::prelude::*;
+
+    #[tokio::test]
+    async fn can_get_expiry() {
+        let (instance, _id, wallet, _wallet2) = get_contract_instance().await;
+
+        let wallet_identity = Identity::Address(Address::from(wallet.address()));
+        let name = String::from("SwaySway");
+
+        register(&instance, &name, 5000, &wallet_identity, &wallet_identity).await;
```

Comment:
The 5000 seems to be used multiple times. I would consider moving this into a `setup` and referencing that variable.